src/Arcus.Messaging.Tests.Integration/MessagePump/Fixture/AssertX.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.Extensions.Logging;
+using Polly;
+using Polly.Retry;
+using Xunit;
+
+namespace Arcus.Messaging.Tests.Integration.MessagePump.Fixture
+{
+    public static class AssertX
```

Comment:
Why AssertX ?  Just wondering :)

Reply:
Because to remove conflict from xUnit `Assert` class. (Oh, how I which we were in F# 😉, with their automatic module merging). We did this on other Arcus repo's too.